app/main.py

```diff
@@ -1,3 +1,14 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    cache_data = {}
+
```

Comment:
add `wraps`

Reply:
you must add like here https://www.geeksforgeeks.org/python-functools-wraps-function/